content/300-accelerate/400-api-reference.mdx

```diff
@@ -34,23 +34,28 @@ The `cacheStrategy` parameter takes an option with the following keys:
 | ------ | ------- | ----- | -------- | ------------------------------------------- |
 | `swr`  | `60`    | `Int` | No       | The stale-while-revalidate time in seconds. |
 | `ttl`  | `60`    | `Int` | No       | The time-to-live time in seconds.           |
+| `tags` | `["user"]` | `String[]` | No | An optional array of strings used to [invalidate](/accelerate/api-reference#accelerateinvalidate) the cache, where each tag may contain only alphanumeric characters and underscores, with a maximum length of 64 characters. |
```

Comment:
Do we have a section that explains why to use each one of these? In a sense comparing them?

Reply:
@mhessdev we have this page https://e87891b9.docs-51g.pages.dev/accelerate/caching, and it touches on how to use each one of those options.

Reply:
Lets change of the section on that page to On-demand cachne invalidation and move it up in line with SWR and TTL